### PR TITLE
Use InfiniteQueries instead of Cursors

### LINKED ISSUE #147: Use InfiniteQueries instead of Cursors
Use InfiniteQueries instead of Cursors so that read minimization and invalidation is easier to handle

### PR DESCRIPTION
**Pull request by JakeSteinebronn**
# Changes
- User Posts feed uses IQ
- User following/ers uses IQ
I promise not all of these lines are significant, I just auto-fix imports and some of the imports were mal-formatted, but sorry for the bloated diff :( 
# Issue ticket number and link
closes #147

### CODE REVIEW

**Comment by abrugal** [comment]
Can you please create an issue for this?

**Comment by JakeSteinebronn** [comment]
done

File: screens/account/SignInOrRegister.tsx [hunk 1]
```diff
@@ -38,8 +38,8 @@ const SignInOrRegister = () => {
   const toast = useToast();
 
   const [formData, setData] = useState<SignInFormData>({
```
Comment (abrugal) [inline screens/account/SignInOrRegister.tsx:40]: Why were the default sign in values changed to this?

**Review by abrugal — APPROVED** [review]
Looks good, just left one comment

File: screens/account/SignInOrRegister.tsx (hunk 1 above)
Comment (JakeSteinebronn) [reply screens/account/SignInOrRegister.tsx:40]: yeah so the local storage doesn't work anymore, so I didn't want to have to type in my credentials every time I wanted to test something. I think @tylerhm is working on a fix for this but in the mean time, I've just been doing this

It's a tester account, so feel free to use it yourself for testing purposes, or change this to your creds if you want to test with your account

**Pull request merged**